<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lançamentos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="dgvDadosLancamento">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>1011</width>
       <height>411</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Codigo </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data da Compra</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vencimento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Credor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ano</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Parc</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Valor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Classificação</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Centro de Custo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cartão</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pesquisar por mes de Vencimento</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPesquisarData">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Executar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExcell">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Excell</string>
     </property>
    </widget>
    <widget class="QComboBox" name="txtPesquisaMes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
